### PR TITLE
First task (create header+cards)

### PR DESCRIPTION
**Pull request by kobekovich**
Hi, @hopole 

Could you take a look please?

Thanks

### CODE REVIEW

File: src/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+class Header extends Component {
+    render() {
+      return (
+        <Navbar bg="info" fixed="top" expand="md">
+          <Navbar.Brand href="#home" className="text-white">NBA-daily</Navbar.Brand>
+          <Navbar.Toggle aria-controls="basic-navbar-nav" />
+          <Navbar.Collapse id="basic-navbar-nav" className="justify-content-center">
+            <Nav className="mr-auto">
+              <Nav.Link href="#teams">Teams</Nav.Link>
```
Comment (hopole) [inline src/Header/Header.js]: render from config

File: src/Cards/Cards.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+class Cards extends Component {
+    render() {
+      return (
+        <Col xs={8} md={6} lg={3}>
+         <Card bg="light" text="dark" className="text-center" border="info" style={{ marginBottom: '10px'}}>
+          <Card.Img variant="top" src={this.props.src} />
+          <Card.Body>
+            <Card.Title className="classTitle">
+             {this.props.name} {this.props.surname}
```
Comment (hopole) [inline src/Cards/Cards.js]: use props distructuring

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,34 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+const cardsList = (
+  <Row className="justify-content-center">
+    <Cards name="Kawhi" surname="Leonard" team="LAC" src="./assets/kawhi.png"/>
```
[the remark this answers is not in the document]
Comment (kobekovich) [reply src/App.js]: Done

File: src/App.js [hunk 4]
```diff
@@ -1,26 +1,34 @@
...
+    <Cards name="Alex" surname="Len" team="ATL" src="./assets/len.png"/>
```
[the remark this answers is not in the document]
Comment (kobekovich) [reply src/App.js]: Done

File: src/App.js [hunk 5]
```diff
@@ -1,26 +1,34 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+const cardsList = (
+  <Row className="justify-content-center">
```
[the remark this answers is not in the document]
Comment (kobekovich) [reply src/App.js]: Done

File: src/App.js [hunk 6]
```diff
@@ -1,26 +1,34 @@
...
+  </Row>
+);
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <Header />
+        <Container>
+          {cardsList}
```
[the remark this answers is not in the document]
Comment (kobekovich) [reply src/App.js]: Done

File: src/Cards/Cards.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Card from 'react-bootstrap/Card';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+
+class Cards extends Component {
```
[the remark this answers is not in the document]
Comment (kobekovich) [reply src/Cards/Cards.js]: Fixed

File: src/App.js [hunk 8]
```diff
@@ -1,26 +1,34 @@
...
+const cardsList = (
+  <Row className="justify-content-center">
+    <Cards name="Kawhi" surname="Leonard" team="LAC" src="./assets/kawhi.png"/>
+    <Cards name="Alex" surname="Len" team="ATL" src="./assets/len.png"/>
+    <Cards name="LeBron" surname="James" team="LAL" src="./assets/lbj.png"/>
+    <Cards name="Svi" surname="Mykhailiuk" team="DET" src="./assets/svi.png"/>
+    <Cards name="Luka" surname="Doncic" team="DAL" src="./assets/luka.png"/>
+    <Cards name="Jason" surname="Tatum" team="BOS" src="./assets/tatum.png"/>
+    <Cards name="Stephen" surname="Curry" team="GSW" src="./assets/curry.png"/>
+    <Cards name="Derrick" surname="Rose" team="DET" src="./assets/rose.png"/>
```
[the remark this answers is not in the document]
Comment (kobekovich) [reply src/App.js]: the logic changed

File: src/Header/Header.js (hunk 1 above)
Comment (kobekovich) [reply src/Header/Header.js]: Done (to check, i'm note sure)

File: src/Cards/Cards.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    render() {
+      return (
+        <Col xs={8} md={6} lg={3}>
+         <Card bg="light" text="dark" className="text-center" border="info" style={{ marginBottom: '10px'}}>
```
[the remark this answers is not in the document]
Comment (kobekovich) [reply src/Cards/Cards.js]: Done

File: src/Header/Header.js (hunk 1 above)
Comment (kobekovich) [reply src/Header/Header.js]: Done

File: src/Cards/Cards.js (hunk 2 above)
Comment (kobekovich) [reply src/Cards/Cards.js]: Done

**Pull request merged**